### PR TITLE
Transfer - Show delayed files

### PR DESCRIPTION
**Pull request by yahavi**
- [x] All [tests](https://github.com/jfrog/jfrog-cli-core#tests) passed. If this feature is not already covered by the tests, I added new tests.
- [x] All [static analysis checks](https://github.com/jfrog/jfrog-cli-core/actions/workflows/analysis.yml) passed.
- [x] This pull request is on the dev branch.
- [x] I used gofmt for formatting the code before submitting the pull request.
-----

Progressbar:
image

Status:
image

### CODE REVIEW

**Review by eyalbe4 — CHANGES_REQUESTED** [review]
Let's add the following comment right after the "Delayed files" value in the --startus display, to explain what "Delayed files" are:

```
De;ayed files:        100 (Files to be transferred last, after all other files.)
```

File: utils/progressbar/transferprogressbarmanager.go [hunk 1]
```diff
@@ -1,34 +1,38 @@
...
-	"time"
+	"github.com/vbauerster/mpb/v7/decor"
 )
 
 const (
-	phase1HeadLine = "Phase 1: Transferring all files in the repository"
-	phase2HeadLine = "Phase 2: Transferring newly created and modified files"
+	phase1HeadLine          = "Phase 1: Transferring all files in the repository"
+	phase2HeadLine          = "Phase 2: Transferring newly created and modified files"
+	delayedFilesContentNote = "Files to be transferred last, after all other files"
```
Comment (eyalbe4) [inline utils/progressbar/transferprogressbarmanager.go:17]: Let's use this const in `status.go`, where this const's value is duplicated.

File: utils/progressbar/transferprogressbarmanager.go [hunk 2]
```diff
@@ -43,13 +47,13 @@ func formatString(emoji, key string, windows bool) string {
 
 func initSProgressBarLabels(windows bool) transferLabels {
 	pbs := transferLabels{}
-	pbs.RetryFailureContentNote = "In Phase 3 and in subsequent executions, we'll retry transferring the failed files."
```
Comment (eyalbe4) [inline utils/progressbar/transferprogressbarmanager.go:46]: Was this line removed intentionally?

**Review by eyalbe4 — CHANGES_REQUESTED** [review]

File: utils/progressbar/transferprogressbarmanager.go (hunk 2 above)
Comment (yahavi) [reply utils/progressbar/transferprogressbarmanager.go:46]: Yes, it is not smart to store const text in a class variable

**Pull request merged**